packages/reports/addon/utils/enums/filter-operations.js

```diff
@@ -1,18 +1,20 @@
 /**
- * Copyright 2017, Yahoo Holdings Inc.
+ * Copyright 2019, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 
-import Ember from 'ember';
+import { A as arr } from '@ember/array';
+import { assert } from '@ember/debug';
+import { typeOf } from '@ember/utils';
+import { get } from '@ember/object';
 import config from 'ember-get-config';
-const { get, assert, typeOf, computed } = Ember;
 
 export default {
   /**
    * @property {Array} _definitions - list of response formats
    */
-  _definitions: computed(function() {
-    let defaultOperations = Ember.A([
+  _definitions() {
```

Comment:
Changed to function because computed property seems to no longer work in a plain object